staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -697,6 +697,9 @@ func shouldSyncUpdatedNode(oldNode, newNode *v1.Node) bool {
 	if oldNode.Spec.ProviderID != newNode.Spec.ProviderID {
 		return true
 	}
+	if oldNode.Annotations[v1.LabelTopologyRegion] != newNode.Annotations[v1.LabelTopologyRegion] {
```

Comment:
Wait a minute, I am looking at a GKE cluster and seeing `v1.LabelTopologyRegion` used as a label, not annotation

Reply:
Also: I am assuming you only bother with validating `v1.LabelTopologyRegion` because it's mutually _inclusive_ with `v1.LabelTopologyZone` (i.e these must be set at the same time, and are not independent of one another)

Reply:
> Wait a minute, I am looking at a GKE cluster and seeing v1.LabelTopologyRegion used as a label, not annotation
Right! I don't know how I missed that. Good catch!

> Also: I am assuming you only bother with validating v1.LabelTopologyRegion because it's mutually inclusive with v1.LabelTopologyZone (i.e these must be set at the same time, and are not independent of one another)

This is true to some extent

But from what I see in the codebase the LabelTopologyZone is used more often then LabelTopologyRegion so I switched to using zone label

Reply:
> But from what I see in the codebase the LabelTopologyZone is used more often then LabelTopologyRegion so I switched to using zone label

I would be fine with adding the region label as well, and triggering an additional re-sync just to make sure.

Reply:
Agree.  Changing region without zone seems weird but we know these are somewhat under-defined.